postgre-client/src/asset_filter_client.rs

```diff
@@ -227,6 +231,10 @@ fn add_filter_clause<'a>(
                     query_builder.push(" OR fungible_tokens.fbt_owner = ");
                     query_builder.push_bind(owner_address);
                     query_builder.push(" ) ");
+                    if !options.show_zero_balance {
```

Comment:
This should be inside the OR clause like `and (asset.owner = xxx or (ft.owner = xxx and ft.balance > 0))`